pkg/ccl/multitenantccl/tenantcostclient/tenant_side.go

```diff
@@ -241,9 +237,15 @@ type tenantSideCostController struct {
 	// avgSQLCPUPerSec is an exponentially-weighted moving average of the SQL CPU
 	// usage per second; used to estimate the CPU usage of a query. It is only
 	// written in the main loop, but can be read by multiple goroutines so is an
-	// atomic.
+	// atomic. It is a Uint64-encoded float64 value.
 	avgSQLCPUPerSec atomic.Uint64
 
+	// writeQPS is the recent global rate of write batches per second for this
+	// tenant, measured across all SQL pods. It is only written in the main loop,
+	// but can be read by multiple goroutines so is an atomic. It is a
+	// Uint64-encoded float64 value.
+	writeQPS atomic.Uint64
```

Comment:
This would be stored in the new `tenant_usage.rates` columns, so we could verify it there. My worry about exporting it from the client as a metric is that I think it could be confusing. It's not write QPS for the SQL client, but instead the _total_ write QPS across all SQL clients. If there is more than one SQL client, then write QPS will be a different number than computing `rate(tenant.sql_usage.write_batches[...])`. Furthermore, different SQL clients could export different numbers, depending on their exact timing of calls to the server - some numbers will be fresher than others.